homeassistant/components/frontend/www_static/polymer/dialogs/more-info-dialog.html

```diff
@@ -8,12 +8,14 @@
 <polymer-element name="more-info-dialog">
 <template>
   <ha-dialog id="dialog" on-core-overlay-open="{{dialogOpenChanged}}">
-    <div>
+    <div class="{{moreInfoTypeClass}}">
       <state-card-content stateObj="{{stateObj}}" style='margin-bottom: 24px;'>
       </state-card-content>
+
       <template if="{{hasHistoryComponent}}">
-        <state-timeline stateHistory="{{stateHistory}}" isLoadingData="{{isLoadingHistoryData}}"></state-timeline>
+        <state-timeline stateHistory="{{stateHistory}}" isLoadingData="{{isLoadingHistoryData}}" hidden?="{{!showHistory}}"></state-timeline>
```

Comment:
Instead of using the `hidden?` attribute we should include this into the `<template if="…">`